<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui::FileLayoutPage</class>
 <widget class="QWidget" name="QtGui::FileLayoutPage">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>File layout</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox { font-weight: bold };</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="forceRewriteCheckBox">
     <property name="text">
      <string>Force rewrite when applying changes
• enforces creation of a temporary/backup file
• enforces use of preferred padding
• causes longer saving-times</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="paddingGroupBox">
        <property name="title">
         <string>Padding</string>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="minPaddingLabel">
             <property name="text">
              <string>Minimum padding</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="minPaddingSpinBox">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> byte</string>
             </property>
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="maxPaddingLabel">
             <property name="text">
              <string>Maximum padding</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="maxPaddingSpinBox">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> byte</string>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>429496729</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="preferredPaddingLabel">
             <property name="text">
              <string>Preferred padding</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="preferredPaddingSpinBox">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> byte</string>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>429496729</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="tagPositionGroupBox">
        <property name="title">
         <string>Preferred tag position</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="tagPosBeforeDataRadioButton">
           <property name="text">
            <string>Befo&amp;re (media) data / at the beginning of the file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tagPosAfterDataRadioButton">
           <property name="text">
            <string>After (&amp;media) data / at the end of the file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tagPosKeepRadioButton">
           <property name="text">
            <string>Keep &amp;previous position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="tagPosForceCheckBox">
           <property name="text">
            <string>Use preferred position even if putting the tags at
another position would prevent rewriting the entire file</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="indexPositionGroupBox">
     <property name="title">
      <string>Index position</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="indexPosBeforeDataRadioButton">
        <property name="text">
         <string>Before (media) data / at the be&amp;ginning of the file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="indexPosAfterDataRadioButton">
        <property name="text">
         <string>After (media) data / at the end of the fi&amp;le</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="indexPosKeepRadioButton">
        <property name="text">
         <string>Keep &amp;previous position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="indexPosForceCheckBox">
        <property name="text">
         <string>Use preferred position even if putting the index at
another position would prevent rewriting the entire file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="NotificationLabel" name="preferredTagPosLabel" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NotificationLabel</class>
   <extends>QWidget</extends>
   <header>gui/notificationlabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>forceRewriteCheckBox</tabstop>
  <tabstop>minPaddingSpinBox</tabstop>
  <tabstop>maxPaddingSpinBox</tabstop>
  <tabstop>preferredPaddingSpinBox</tabstop>
  <tabstop>tagPosBeforeDataRadioButton</tabstop>
  <tabstop>tagPosAfterDataRadioButton</tabstop>
  <tabstop>tagPosKeepRadioButton</tabstop>
  <tabstop>tagPosForceCheckBox</tabstop>
  <tabstop>indexPosBeforeDataRadioButton</tabstop>
  <tabstop>indexPosAfterDataRadioButton</tabstop>
  <tabstop>indexPosKeepRadioButton</tabstop>
  <tabstop>indexPosForceCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
